reference/admin.yaml

```diff
@@ -7709,6 +7709,158 @@ components:
+        category_id:
+          type: string
+          description: |
+            Category ids of the expenses.
+          example: in.(60506)
+        project_id:
+          type: string
+          description: |
+            Project ids of the expenses.
+          example: in.(261155)
+        cost_center_id:
+          type: string
+          description: |
+            Cost center ids of the expenses.
+          example: in.(7475)
+        currency:
+          type: string
+          description: |
+            Currency of the expenses.
+          example: USD
+        vehicle_type:
+          type: string
+          description: |
+            Vehicle type of the expenses.
+          example: ELECTRIC_CAR
+        employee_id:
+          $ref: '#/components/schemas/employee_id'
+    expense_policy_individual_desired_states:
+      type: object
+      properties:
+        rule:
+          $ref: '#/components/schemas/expense_policy_rule_minimal'
+        amount:
+          allOf:
+            - $ref: '#/components/schemas/amount'
+          description: |
+            Amount calculated by policy to be applied to the expense.
+          example: 500
+        violating_expenses_query_params:
+          $ref: '#/components/schemas/expense_policy_violating_expenses_query_params'
+    expense_policy_final_desired_state:
+      type: object
+      properties:
+        expense_id:
+          type: string
+          description: |
+            This represents the id of the expense.
+          example: txabsdfSYqd
+        flag:
+          type: boolean
+          description: |
+            If it is set to true, then the expense will be flagged.
+          example: true
+        amount:
+          allOf:
+            - $ref: '#/components/schemas/amount'
+          description: |
+            Amount calculated by policy to be applied to the expense.
+          example: 500
+        is_receipt_mandatory:
+          type: boolean
+          description: |
+            If this is set to true, then the expense requires receipt.
+          example: true
+        add_approver_user_ids:
+          type: array
+          items:
+            type: string
+          description: >
+            List of approver user ids that will be added to the report that
+            contains this expense.
+          example:
+            - usBdsdfscl
+            - usLSsdfwqt
+        remove_employee_approver1:
+          type: boolean
+          description: >
+            If this is set to true, then the employee's first approver will be
+            removed from the report that contains this expense.
+          example: true
+        run_status:
+          type: string
+          description: |
+            Whether policy ran successfully or not.
+          example: SUCCESS
+        run_summary:
+          type: array
+          items:
+            type: string
+          description: |
+            List of action summary that needs to happen on the expense.
```

Comment:
List of human-readable action summaries that the {policy calculation} suggests for this expense.

Reply:
all roles